SeaPublicWebsite/ExternalServices/EpbEpc/EpbEpcApi.cs

```diff
@@ -152,14 +153,67 @@ private async Task<string> RequestTokenIfNeeded()
 
         private static HeatingType? GetHeatingTypeFromEpc(EpbEpcAssessmentDto epc)
         {
-            return epc.MainFuelType switch
+            // Gas boiler check
+            if (epc.MainFuelType.Equals("20") ||
+                epc.MainFuelType.Equals("26") ||
+                epc.MainFuelType.Contains("mains gas", StringComparison.OrdinalIgnoreCase))
```

Comment:
We conduct checks for both numbers and textual correspondences. Confirmed the numbers from Kevin and approved the content with Brian.